src/examples/src/widgets/select/Basic.tsx

```diff
@@ -3,18 +3,20 @@ import Select from '@dojo/widgets/select';
 import icache from '@dojo/framework/core/middleware/icache';
 
 const factory = create({ icache });
-const options = ['cat', 'dog', 'fish'];
+const options = [{ value: 'cat' }, { value: 'dog' }, { value: 'fish' }];
 
 export default factory(function Basic({ middleware: { icache } }) {
 	return (
-		<Select
-			label="Simple Native Select"
-			useNativeElement={true}
-			value={icache.get('value')}
```

Comment:
Why did you take the value property out of the basic example?

Reply:
I was using this for dev, I will move that out into different examples